.github/workflows/unit_tests.yml

```diff
@@ -45,19 +51,37 @@ jobs:
       - name: Run unit tests
         run: ./gradlew test coverage
 
-      - name: Save PR number to file
-        run: echo ${{ github.event.number }} > PR_NUMBER.txt
-
-      - name: Archive PR number
-        uses: actions/upload-artifact@v4
-        with:
-          name: pr_number
-          path: PR_NUMBER.txt
-
       - name: Archive reports
         uses: actions/upload-artifact@v4
         with:
           name: unit_test_reports
           path: |
             **/build/**/jacoco*.xml
             **/build/classes/
+  sonarqube:
```

Comment:
Does this job not also need the other steps that other jobs have, like 'Mask secrets', 'Check out repository', 'Set Environment Variables', 'Rebase', 'Set up Java', 'Setup Gradle' ?

Reply:
Yeah, that was my bad, I forgot to add those steps! Updated and now the job is working properly.